docker/Dockerfile.exec

```diff
@@ -12,20 +12,14 @@ RUN go get -v github.com/readium/readium-lcp-server/lcpencrypt
 ## Final image
 ###############################################################################
 
-FROM phusion/baseimage:focal-1.0.0
+FROM ghcr.io/REPOSITORY_OWNER/docker-base:TAG
```

Comment:
Once you combine the dockerfiles, you don't have to template any names like this.